README.md

```diff
@@ -1,8 +1,8 @@
 # BrainySnake
 ![brainySnakePic](/Dokumentation/brainySnake.png)
 
-Brainysnake ist ein Snake-Spiel bei welchem ihr eure Schlange nicht aktiv selbst steuert, sondern über einen Bewegungsalgorithmus vorher festlegt, wie eure Schlange sich verhalten soll.
-Dabei werden pro Spiel mehrere autonome Schlangen in einem Level geladen. Dann beginnt der Kampf um den Sieg. Gewonnen hat, wer nach 1000 Zügen (Anzahl der Züge ist konfigurierbar) noch lebt und die meisten Punkte hat.
+BrainySnake is a game based on the all time classic game "Snake". In BrainySnakes you don't actively control your snake but instead program its behaviour so it follows certain patterns.
```

Comment:
> In BrainySnakes you don't actively control your snake but instead program its behaviour so it follows certain patterns. 


Liest sich etwas komisch.
Vorschlag: But in BrainySnake you can't control the snake with your keyboard, instead you develop your own AI to do so!

Reply:
"Ghostmode" und "Snake" sind in diesem Kontext Eigennamen und deshalb groß geschrieben.
Ich finde meine Formulierung eloquenter, aber wenn du meinst, dass deine Formulierung vielleicht leichter zu verstehen ist, können wir sie gerne übernehmen :)

Reply:
Das ist nur ein Vorschlag, BrainySnakes sollte aber noch zu BrainySnake geändert werden. 
Ob "Snake" ein Eigenname ist, bin ich mir gerade nicht sicher.